command/snapshot/save/snapshot_save.go

```diff
@@ -23,17 +26,26 @@ func New(ui cli.Ui) *cmd {
 }
 
 type cmd struct {
-	UI    cli.Ui
-	flags *flag.FlagSet
-	http  *flags.HTTPFlags
-	help  string
+	UI                 cli.Ui
+	flags              *flag.FlagSet
+	http               *flags.HTTPFlags
+	help               string
+	appendFileNameFlag flags.StringValue
+}
+
+func (c *cmd) getAppendFileNameFlag() *flag.FlagSet {
+	fs := flag.NewFlagSet("", flag.ContinueOnError)
+	fs.Var(&c.appendFileNameFlag, "append-filename", "Append filename flag takes two possible values. "+
```

Comment:
I guess `append-to-filename` sounds better semantically. WDYT?

Reply:
This was given by @david-yu . So kept it that way. I am fine with both.

Reply:
https://hashicorp.slack.com/archives/C055CK3BMR8/p1693461179291959?thread_ts=1693397494.647169&cid=C055CK3BMR8